website/src/types.d.ts

```diff
@@ -6,3 +6,4 @@
  */
 
 /// <reference types="@docusaurus/plugin-ideal-image" />
+/// <reference types="@types/gtag.js" />
```

Comment:
not very satisfied with this setup atm, as it puts the burden to the site owner to reference the appropriate packages, but good enough to move on

What I wish is to have the type auto reference itself if the preset-classic is installed (since it contains the gtag plugin, even though it might be disabled).

I'd like to have references: preset => plugin gtag => `@types/gtag.js`

For now, if I add such a ref in `index.ts`, it gets stripped out from the `.d.ts` file, unfortunately.

@Josh-Cena I'll merge as it it but if you have any idea to make that site ref un-necessary that would be cool.